### ISSUE TITLE
My Inbox is gone after system reinstall

### ISSUE THREAD

**Issue by antimech** [alainm23/planify#479]
**Describe the bug**
My inbox tasks continually disappear.

**To Reproduce**
Steps to reproduce the behaviour:
1. Start Planner for the first time (create DB), fill it out, fill Inbox and backup the DB file;
2. Copy backup to another machine and start Planner there;
3. Change DB location in settings to your backup;
4. See Inbox is gone, everything else remain;
5. Try to fill Inbox again;
6. Try to open it a couple of times, after reboot for example;
7. See it's gone again.

**Expected behaviour**
I'm clearly not expecting to lose all my inbox tasks.

**Desktop:**
 - OS: Linux Mint 20 Cinnamon
 - Version: v2.4.6 (Flathub)

**Additional context**
I was backing up my Planner DB, just like KeePassXC and preparing for system reinstall. After installing fresh system, installing Planner and changing DB path to my home directory with backup I saw that all my Inbox tasks are gone (a lot of them). I started to fill it again but after some time I saw that it's gone again.

**Comment by antimech** [comment]
I've found something interesting. When I launched search (Ctrl + F) I was searching for some of my tasks and surprisingly I saw one of my tasks that was gone, I clicked on it and entered Inbox with all my tasks that was gone but the Inbox is not visible on my side panel. I mean it's visible but that Inbox is empty, so it looks like there are two Inbox now.

Or Planner consider this Inbox as a invisible project:
![image](https://user-images.githubusercontent.com/13885653/88881806-25b52080-d252-11ea-8ac1-11e8d9c995c7.png)

I've moved all my task to the "real" Inbox and removed the "fake" Inbox project, it fixed my problem temporarily.
